### DISCUSSION TITLE
Does plausible write any logfiles?

### DISCUSSION THREAD

**Discussion by grobmeier** [plausible/analytics#1897] [Self-Hosted Support] [answered]
Question says it all, I would love to look into the logs to learn more about issues from Plausible

**Comment by Vigasaurus** [comment, marked as answer]
You can view the `docker-compose logs` - as far as I know that's the only user-facing log surface (you could theoretically access the journals for postgres, clickhouse, and maybe smtp in their respective containers though).

**Reply by grobmeier** [reply]
Thanks. Those logs are not really "exhaustive", but already help a bit. I have seen I could integrate Sentry by using those logs, so thanks for the pointer
